src/Schema/ObjectSchema.php

```diff
@@ -18,6 +21,7 @@ public function __construct(
         public readonly array $properties,
         public readonly array $requiredFields = [],
```

Comment:
Not sure if we should spin through the properties and see which are nullable here? That may break backwards compatibility? Not entirely sure.